modules/module-info.php

```diff
@@ -404,12 +404,29 @@ function jetpack_photon_more_link() {
 
 function jetpack_photon_more_info() {
 	esc_html_e(
-		"Your images are automatically optimized for different display resolutions to serve the best
-		possible image quality. We also cache and serve them from our fast, global network (CDN)."
+		'Jetpack will optimize your images and serve them from the server location nearest
+		to your visitors. Using our global \'content delivery network\' will boost the loading speed of your site.'
 		, 'jetpack' );
 }
 add_action( 'jetpack_module_more_info_photon', 'jetpack_photon_more_info' );
 
+/**
+ * Lazy Images
+ */
+function jetpack_lazy_images_more_link() {
+	echo 'https://jetpack.com/support/lazy-images/';
+}
+add_action( 'jetpack_learn_more_button_lazy-images', 'jetpack_lazy_images_more_link' );
+
+function jetpack_lazy_images_more_info() {
+	esc_html_e(
+		'Normally, opening a web page causes it to load every image it contains, even those
+		that a reader hasn\'t yet scrolled down to see. "Lazy loading" means that only the images
+		actually visible on the screen will load.'
```

Comment:
Maybe we don't need to explain lazy loading and instead what this toggle do?
I'd change
`"Lazy loading" means that only the images actually visible on the screen will load.`
to
`Enable this to load the images visible to users as they scroll down`

Reply:
I would preserve the way @MichaelArestad has written it ("Lazy loading" means...) since it's a term not many Jetpack users will be familiar with.

Or we could opt for best of both worlds:

> Enable this for "lazy loading" behavior, where only the images actually visible on the screen will load.